Source/main.cpp

```diff
@@ -345,7 +349,7 @@ void main_main ()
     }
 
     // Setup stopping criteria
-    Real Tol = 1.0e-19;
+    Real Tol = 1.0e-18;
```

Comment:
Tol=1.e-18 is much to small; 1e-10 should be ok.  And this should be parsable via the inputs file, please incorporate that option.